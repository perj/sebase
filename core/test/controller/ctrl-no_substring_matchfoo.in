no_substring_matchfoo
