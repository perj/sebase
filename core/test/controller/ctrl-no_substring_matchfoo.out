{
 "message": "Forbidden (/no_substring_matchfoo)",
 "status": "403"
}
